randovania/game_connection/builder/dolphin_connector_builder.py

```diff
@@ -11,12 +12,62 @@
 
 
 class DolphinConnectorBuilder(PrimeConnectorBuilder):
+    dolphin_cmd: str
+    dolphin_comm: str
+    dolphin_child: Popen
+
+    def __init__(self, dolphin_cmd: str = "", dolphin_comm: str = ""):
+        super().__init__()
+        self.dolphin_cmd = dolphin_cmd
+        self.dolphin_comm = dolphin_comm
+        self.dolphin_child = None
+
+    @property
+    def pretty_text(self) -> str:
+        if self.dolphin_comm:
+            if self.dolphin_cmd:
+                return f"{super().pretty_text}: run '{self.dolphin_cmd}', attach to '{self.dolphin_comm}'"
+            else:
+                return f"{super().pretty_text}: autodetect as '{self.dolphin_comm}'"
+        elif self.dolphin_comm:
```

Comment:
Smells like unreachable code: 
```
if self.dolphin_comm:
...
elif self.dolphin_comm:
```

Reply:
Yep, that second one was meant to be `dolphin_cmd`. Thanks.